__init__.py

```diff
@@ -200,6 +238,96 @@ def handle_show_debug(self, message):
                                       {"enabled": enabled}))
         # TODO: Handle this event DM
 
+    @intent_handler("classic_mycroft.intent")
```

Comment:
Consider an Adapt intent with regex so this is extensible to configurable voice names

Reply:
Open to it, I just need to learn Adapt 😉

Reply:
adapt regex doesnt work very well and doesn't make it configurable? it will still capture anything, but has more side effects (such as entity names being shared across skills, no skill_id munging)

I strongly recommend against adapt regexes in general, but feel like i'm missing something in the suggestion here

Reply:
@NeonDaniel ?

Reply:
I was thinking about something like the [Language Intents](https://github.com/NeonGeckoCom/skill-user_settings/blob/dd2cb92e5bfaaef3061bbb4db3398fd6ec4a45de/__init__.py#L793-L795). It looks like I did end up with some [manual regex parsing there](https://github.com/NeonGeckoCom/skill-user_settings/blob/dd2cb92e5bfaaef3061bbb4db3398fd6ec4a45de/__init__.py#L991) though; I'm not sure if that's for the Adapt or Padatious handler..

My concept was an intent like:

```python
IntentBuilder("ChangePersona").require("change").require("rx_personality")
```

change.voc:
```
become
change to
```
personality.rx:
```
(become|to) (?P<rx_personality>.*)(| personality| persona)
```

Then there's one intent and we match `rx_personality` to one of the supported WW/TTS combos instead of having one intent for each option

Reply:
@NeonDaniel I'm a little wary of making much of a change if @JarbasAl is saying it doesn't work well, but if we're following an established pattern that has been working fine, I can make the change. Would you like me to?

Reply:
If this is working as-is, then I'm fine with leaving well enough alone